library/lua/dfhack.lua

```diff
@@ -215,10 +240,12 @@ local function print_element(k, v)
     dfhack.println(string.format("%-23s\t = %s", tostring(k), tostring(v)))
 end
 
+---@param table table
```

Comment:
could also be a c++ vector. do we want to draw that disctinction?

Reply:
If there is no special behaviour there I think it's fine not to include, if it quacks like a table...